### PR TITLE
[1LP][WIPTEST] CandU to widgetastic

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Convert CandU to widgetastic, update tests cases
{{pytest: -v -k test_utilization_metrics.py --long-running}}
Some tests failed not due to widhetastic conversion, as C and U switches work with ec2 provider.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 741fa96b1fff622360f59a9da69ddba8debd68eb:

`cfme/configure/configuration/region_settings.py`:
- :heavy_exclamation_mark: Line 481:56: **W292** *no newline at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/configure/configuration/region_settings.py [hunk 1]
```diff
@@ -428,3 +428,72 @@ class MapTagsEdit(CFMENavigateStep):
...
+            self.all_clusters_cb.is_displayed
+        )
+
+
+class CANDUCollection(Navigatable):
+    """ Class represents a C and U in CFME UI """
+    def __init__(self, appliance=None):
+        Navigatable.__init__(self, appliance=appliance)
+
+    def _enable_disable(self, anable=True, reset=False):
```
Comment (quarckster) [inline cfme/configure/configuration/region_settings.py]: I believe a typo is here.

Comment (rlbabyuk) [inline cfme/configure/configuration/region_settings.py]: corrected

File: cfme/configure/configuration/region_settings.py [hunk 2]
```diff
@@ -428,3 +428,72 @@ class MapTagsEdit(CFMENavigateStep):
...
+            Args:
+                enable: Switches states, 'True'- enable
+                reset: Reset changes, default is 'False' - changes will not be reset
+        """
+        view = navigate_to(self, 'Details')
+        view.fill({
+            'all_clusters_cb': enable,
+            'all_datastores_cb': enable
+        })
+        if reset:
```
Comment (rlbabyuk) [inline cfme/configure/configuration/region_settings.py]: no, I need to change the values as reset button is inactive without any changes done

**Comment by rlbabyuk** [comment]
Tests passed for ec2 and rhos7-qa, fails only rhv41(C and U not gathering, incorrect creds)

File: cfme/configure/configuration/region_settings.py [hunk 3]
```diff
@@ -428,3 +428,72 @@ class MapTagsEdit(CFMENavigateStep):
...
+            self.all_clusters_cb.is_displayed
+        )
+
+
+class CANDUCollection(Navigatable):
+    """ Class represents a C and U in CFME UI """
+    def __init__(self, appliance=None):
+        Navigatable.__init__(self, appliance=appliance)
+
+    def _enable_disable(self, enable=True, reset=False):
```
Comment (jkrocil) [inline cfme/configure/configuration/region_settings.py]: Let's rename this to `_set_state` instead of `_enable_disable`, please.

**Pull request merged**